src/sentry/experiments/manager.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+import six
+
+
+class ExperimentManager(object):
+    def __init__(self):
+        self._experiments = {}
+
+    def add(self, experiment, param, log_exposure=True):
+        self._experiments[experiment.__name__] = {
+            'experiment': experiment, 'param': param, 'log_exposure': log_exposure}
```

Comment:
not sure if this is the best way for this but wanted to make it flexible to take different param names and `log_exposure` values. Also, need to make `param` generalizable when we get to the point when multiple `params` are assigned within an experiment.

Reply:
let's remove the log_exposure param and always pass false in the `all()` method. it doesn't make much sense to be logging exposure when this is being called within an organization-serializer context (serializing an org should be a side-effect free action)